variables.md

```diff
@@ -10,6 +10,7 @@ Variable        | Type      | Default value | Details
 ADDONS          | string    |               | Comma separated list of addons to be added using DVD. Also used to indicate addons in the SUT.
 ADDONURL        | string    |               | Comma separated list of addons. Includes addon names to get url defined in ADDONURL_*. For example: ADDONURL=sdk,we ADDONURL_SDK=https://url ADDONURL_WE=ftp://url
 ADDONURL_*      | string    |               | Define url for the addons list defined in ADDONURL
+ASSERT_Y2LOGS   | boolean   | false         | If set to true, we will parse YaST logs after installation and fail test suite in case unknown errors were detected.
```

Comment:
Any specific reason why you decided for a dedicated scenario and not the existing `installer_extended` scenario with the test variable `INSTALLER_EXTENDED_TEST`?

Reply:
I don't introduce anything new, I've just added variable description.

Reply:
It could be confusing mix both scenarios due to `yast2_failures` will be most of the time failing and `installer_extended` we will be expecting to be green most of the time, but I guess it could be discussed after this change.

Reply:
I also would not mix those 2 scenarios, at least not now.